tests/h/tasks/indexer_test.py

```diff
@@ -61,6 +62,18 @@ def test_it(self, search_index):
         search_index.sync.assert_called_once_with("test_queue")
 
 
+class TestReportSyncAnnotationsQueueLength:
+    def test_it(self, newrelic, search_index):
+        indexer.report_sync_annotations_queue_length()
+
+        search_index._queue.count.assert_called_once_with(
+            ["storage.create_annotation", "storage.update_annotation"]
+        )
+        newrelic.agent.record_custom_metric.assert_called_once_with(
+            Any.string(), search_index._queue.count.return_value
```

Comment:
:+1: Good strategic vagueness here. Lets us change our mind without changing the code. This would sadly have to go if we started reporting more than one though.